agent/php_nrini.c

```diff
@@ -1891,6 +1891,46 @@ static PHP_INI_MH(nr_custom_events_max_samples_stored_mh) {
   return SUCCESS;
 }
 
+#define DEFAULT_WORDPRESS_HOOKS_OPTIONS "all_callbacks"
+static PHP_INI_MH(nr_wordpress_hooks_options_mh) {
+  nrinistr_t* p;
+
+#ifndef ZTS
+  char* base = (char*)mh_arg2;
+#else
+  char* base = (char*)ts_resource(*((int*)mh_arg2));
+#endif
```

Comment:
We don't need to worry about ZTS, right?

Reply:
Removed in 97edd07795b66bdcc74920fd321f18e750391f7e.